src/main/index.ts

```
@@ -196,26 +201,53 @@ function readDirectoryRecursive(dirPath: string): any[] {
+    const node: any = {
+      id: fullPath,
       name,
       type: isDirectory ? 'folder' : 'file',
       path: fullPath,
       children: isDirectory ? readDirectoryRecursive(fullPath) : undefined
     }
+
+    if (isFile) {
+      try {
+        node.content = readFileSync(fullPath, 'utf-8')
+      } catch (e) {
```

ESLint

[error] 'e' is defined but never used.

(@typescript-eslint/no-unused-vars)